src/cli.rs

```diff
@@ -48,40 +55,37 @@ impl Config {
     pub fn new() -> Result<Self> {
         let matches = build_cli().get_matches();
 
-        let true_color = match matches.value_of("true-color") {
-            Some("always") => true,
-            Some("never") => false,
-            Some("auto") => is_truecolor_terminal(),
+        let true_color: String = matches.get_one("true-color").cloned().unwrap();
+        let true_color = match true_color.as_str() {
+            "always" => true,
+            "never" => false,
+            "auto" => is_truecolor_terminal(),
             _ => unreachable!(),
         };
 
-        let no_bold = matches.is_present("no-bold");
-        let no_merges = matches.is_present("no-merges");
-        let no_color_palette = matches.is_present("no-palette");
-        let print_languages = matches.is_present("languages");
-        let print_package_managers = matches.is_present("package-managers");
-        let iso_time = matches.is_present("isotime");
-        let show_email = matches.is_present("email");
-        let include_hidden = matches.is_present("hidden");
-
-        let output = matches
-            .value_of("output")
-            .map(SerializationFormat::from_str)
-            .transpose()?;
-
-        let fields_to_hide: Vec<String> = if let Some(values) = matches.values_of("disable-fields")
-        {
-            values.map(String::from).collect()
-        } else {
-            Vec::new()
-        };
-
-        let disabled_fields = InfoFieldOff::new(fields_to_hide)?;
-
-        let art_off = match matches.value_of("show-logo") {
-            Some("always") => false,
-            Some("never") => true,
-            Some("auto") => {
+        let no_bold = matches.get_one("no-bold").copied().unwrap();
```

Comment:
```suggestion
        let no_bold = matches.contains_id("no-bold");
```

Maybe we could use `contains_id` instead for this kind of boolean flag and drop the `action(ArgAction::SetTrue)` that was added to the flag definition ?

Reply:
:thinking: Yeah I'm a bit conflicted about this one. `contains_id` is much simpler, but `get_one` is more consistent with other args. Note: I'm a bit biased since `SetTrue` is very similar to Python's `argparse` library, which I've used a lot :laughing: